docs/content/direct/get-started.md

```diff
@@ -1,32 +1,23 @@
 # KubeStellar Quickstart Setup
 
-This Quick Start outlines step 1 and shows a concrete example of steps 2--7 in the [Installation and Usage outline](usage-basics.md). In this example you will create three new `kind` clusters to serve as your KubeFlex hosting cluster and two WECs.
+This Quick Start outlines step 1, shows a concrete example of steps 2--7 in the [Installation and Usage outline](usage-basics.md), and forwards you to one example of the remaining steps. In this example you will create three new `kind` clusters to serve as your KubeFlex hosting cluster and two WECs.
 
   1. Before you begin, prepare your system (get the software prerequisites)
   2. Create the KubeFlex hosting cluster and Kubestellar core components
   3. Create and register two WECs.
+  4. Use KubeStellar to distribute a Deployment object to the two WECs.
 
 ---
 ## Before You Begin
```

Comment:
Can't we just call it "Install pre-reqs" and move the other paragraph to a separate "Cleanup" section ?

Reply:
The paragraph about configuring `kind` to work with more than two clusters is not a matter of cleanup.

Reply:
Sorry, My comment was on the wrong line :(     
I meant change the "Before you begin" to:  Cleanup and install pre-reqs   and also change the list item from "Before you begin, prepare your system (get the software prerequisites)"   to something short (maybe also Cleanup and install pre-reqs)

Reply:
Revised.